pipeline/errors/error_redirect.go

```diff
@@ -66,3 +68,18 @@ func (a *ErrorRedirect) Config(config json.RawMessage) (*ErrorRedirectConfig, er
 func (a *ErrorRedirect) GetID() string {
 	return "redirect"
 }
+
+func (a *ErrorRedirect) RedirectURL(r *http.Request, c *ErrorRedirectConfig) string {
+	if c.UrlParam == "" {
```

Comment:
CI fails because of this
```suggestion
	if c.URLParam == "" {
```